components/Footer/Footer.js

```diff
@@ -0,0 +1,247 @@
+            font-weight: bold;
+            margin-bottom: 20px;
+          }
+          
+          .footer-distributed .footer-icons {
+            margin-top: 25px;
+          }
+          
+          .footer-distributed .footer-icons a {
+            display: inline-block;
+            width: 35px;
+            height: 35px;
+            cursor: pointer;
+            background-color: #33383b;
+            border-radius: 2px;
+          
+            font-size: 20px;
+            color: #ffffff;
+            text-align: center;
+            line-height: 35px;
+          
+            margin-right: 3px;
+            margin-bottom: 5px;
+          }
+          
+          /* If you don't want the footer to be responsive, remove these media queries */
+          
+          @media (max-width: 880px) {
+            .footer-distributed {
+              font: bold 14px sans-serif;
+            }
+          
+            .footer-distributed .footer-left,
+            .footer-distributed .footer-center,
+            .footer-distributed .footer-right {
+              display: block;
+              width: 100%;
+              margin-bottom: 40px;
+              text-align: center;
+            }
+          
+            .footer-distributed .footer-center i {
+              margin-left: 0;
+            }
+          }
+          
+        `}</style>
+        <footer className="footer-distributed">
+          <div className="footer-left">
+            <h3>
+              UPAY<span>logo</span>
+            </h3>
+
+            <p className="footer-links">
+              <Link href="/">
+                <a>Home</a>
+              </Link>
+              ·
+              <Link href="/about">
+                <a>About</a>
+              </Link>
+            </p>
+
+            <p className="footer-company-name">UPAY NGO &copy; 2016</p>
+          </div>
+
+          <div className="footer-center">
+            <div>
+              <i className="fa fa-map-marker" />
+              <p>
+                <span>205, Honey Sagar Apartments,</span>
+                <span>Wardhman Nagar Square,</span>
+                Nagpur, 440008, Maharashtra
+              </p>
+            </div>
+
+            <div>
+              <i className="fa fa-phone" />
+              <p>7030735531</p>
+            </div>
+
+            <div>
+              <i className="fa fa-envelope" />
+              <p>
+                <a href="mailto:info@upay.org.in">info@upay.org.in</a>
+              </p>
+            </div>
+          </div>
+
+          <div className="footer-right">
+            <p className="footer-company-about">
+              <span>About the company</span>
+              The foundation of development for every society is the education
+              of its youth. Keeping this in mind, an NGO UPAY (Underprivileged
+              Advancement by Youth), was established in May 2010 by a group of
+              young engineers from IITs and NITs.
+            </p>
+
```

Comment:
Would it be possible to break the Footer component into smaller components? Anything beyond 150 lines of code seems excessive enough to read.

Reply:
ok, working on it!